src/core/Akka.Persistence/Snapshot/SnapshotStore.cs

```diff
@@ -235,17 +251,20 @@ private Exception TryUnwrapException(Exception e)
         /// </summary>
         /// <param name="metadata">Snapshot metadata.</param>
         /// <param name="snapshot">Snapshot.</param>
-        /// <returns>TBD</returns>
-        protected abstract Task SaveAsync(SnapshotMetadata metadata, object snapshot);
+        /// <param name="cancellationToken"><see cref="CancellationToken"/> used to signal cancelled snapshot operation</param>
+        protected abstract Task SaveAsync(
+            SnapshotMetadata metadata,
+            object snapshot,
+            CancellationToken cancellationToken);
```

Comment:
[Breaking change] `Akka.Persistence` codes that are being wrapped inside a `CircuitBreaker` will now take a `CancellationToken` as a signal that the operation has been timed out by the `CircuitBreaker`.